examples/wildcard-tls-certificate/README.md

```diff
@@ -45,31 +45,26 @@ spec:
               number: 80
  ```
 
-`bar-ingress` from the namespace `bar-namespace`:
+`bar` VirtualServer from the namespace `bar`:
 
 ```yaml
-apiVersion: networking.k8s.io/v1
-kind: Ingress
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
 metadata:
-  name: bar-ingress
-  namespace: bar-namespace
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
+  name: bar
+  namespace: bar
 spec:
+  host: bar.example.com
   tls:
-  - hosts:
-    - bar.example.com
-  rules:
-  - host: bar.example.com
-    http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
-          service:
-            name: bar-service
-            port:
-              number: 80
+    secret: "" 
+  upstreams:
+  - name: bar 
+    service: bar-service
+    port: 80
+  routes:
+  - path: /
+    action:
+      pass: bar
 ```
 
-Because we don't reference any TLS secret in the `tls` section (there is no `secretName` field) in both Ingress resources, NGINX will use the wildcard secret specified in the `-wildcard-tls-secret` command-line argument.
+Because we don't reference any TLS secret in the resources above -- there is no `secret` field  in the `tls` section of the Ingress resource and the `secret` field is empty in the VirtualServer -- NGINX will use the wildcard secret specified in the `-wildcard-tls-secret` command-line argument.
```

Comment:
```suggestion
Because we don't reference any TLS secret in the resources above -- there is no `secret` field in the `tls` section of the Ingress resource and the `secret` field is empty in the VirtualServer -- NGINX will use the wildcard secret specified in the `-wildcard-tls-secret` command-line argument.
```